src/vocabularyChecker/detectWrongSyntax.test.ts

```diff
@@ -4,88 +4,88 @@ import {collectAllInvalidWords, createErrorMessages} from "./detectWrongSyntax";
-	const wrongWords = getInvalidWords(`This is just a really long String to test some extreme case so
-the next letters will only be some copies of the letter a.
-aaaaaaaaaaaaaaaaaaaaaaaaaaa`);
-	expect(wrongWords).toEqual(["This", "is", "just", "a", "really", "long", "String", "to", "test",
-	"some", "extreme", "case", "so", "the", "next", "letters", "will", "only", "be", "copies",
+	it("Works for long strings", () => {
+		const wrongWords = getInvalidWords(`This is just a really long String to test some extreme case so
+		the next letters will only be some copies of the letter a.
+		aaaaaaaaaaaaaaaaaaaaaaaaaaa`);
+		expect(wrongWords).toEqual(["This", "is", "just", "a", "really", "long", "String", "to", "test",
+		"some", "extreme", "case", "so", "the", "next", "letters", "will", "only", "be", "copies",
 		"of", "letter", "aaaaaaaaaaaaaaaaaaaaaaaaaaa"]);
-});
+	});
 
-it("Recognizes the words correctly", () => {
+	it("Recognizes the words correctly", () => {
 		const result = getInvalidWords("Hallo Welt, fu. foo, bar");
 		expect(result).toEqual(["Hallo", "Welt", "foo"]);
-});
+	});
 
-it("Seperates words correctly along dots", () => {
-	const result = getInvalidWords("hello.world");
-	expect(result).toEqual(["hello", "world"]);
-});
+	it("Seperates words correctly along dots", () => {
+		const result = getInvalidWords("hello.world");
+		expect(result).toEqual(["hello", "world"]);
+	});
 }),
 
 describe("logMapElement", () => {
-it("Logs a map element correctly", () => {
-	const issues = logMapElement([0, 4], `test`);
-	expect(issues).toEqual({message: "test an Stelle 0,4 ist ein unerlaubtes Wort! \n",
-	position: {fromIndex: 0, toIndex: 4}});
-});
+	it("Logs a map element correctly", () => {
+		const issues = logMapElement([0, 4], `test`);
+		expect(issues).toEqual({message: "test an Stelle 0,4 ist ein unerlaubtes Wort! \n",
+		position: {fromIndex: 0, toIndex: 4}});
+	});
 
-it("Logs a empty String correctly", () => {
-	const issues = logMapElement([0, 0], ``);
-	expect(issues).toEqual({message: " an Stelle 0,0 ist ein unerlaubtes Wort! \n",
-	position: {fromIndex: 0, toIndex: 0}});
-});
+	it("Logs a empty String correctly", () => {
+		const issues = logMapElement([0, 0], ``);
+		expect(issues).toEqual({message: " an Stelle 0,0 ist ein unerlaubtes Wort! \n",
+		position: {fromIndex: 0, toIndex: 0}});
+	});
 
-it("Logs a String containing multiple words correctly", () => {
-	const text = `test Test`;
-	const issues = logMapElement([0, text.length], text);
-	expect(issues).toEqual({message: text + " an Stelle 0," + text.length + " ist ein unerlaubtes Wort! \n",
-	position: {fromIndex: 0, toIndex: text.length}});
-});
+	it("Logs a String containing multiple words correctly", () => {
+		const text = `test Test`;
+		const issues = logMapElement([0, text.length], text);
+		expect(issues).toEqual({message: text + " an Stelle 0," + text.length + " ist ein unerlaubtes Wort! \n",
+		position: {fromIndex: 0, toIndex: text.length}});
```

Comment:
Please separate object literals onto multiple lines, so it's easier to read:

```typescript
expect(issues).toEqual({
  message: text + " an Stelle 0," + text.length + " ist ein unerlaubtes Wort! \n",
  position: {
    fromIndex: 0,
    toIndex: text.length,
  },
});
```

Please notice the `,` after every property (including the last one). This is so git (which diffs line-by-line) doesn't count added commas in the line before as changed lines. Common convention.